apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net.Tests/Log4NetTest.cs

```diff
@@ -179,6 +179,52 @@ private LoggingEvent CreateLog(Level level, string msg, int secondsOfs = 0)
             .ToString(GoogleStackdriverAppender.s_lostDateTimeFmt);
 
         [Fact]
+        public async Task PostActivationChangeThrow()
+        {
+            await RunTestWorkingServer(appender =>
+            {
+                Assert.Throws<InvalidOperationException>(() => { appender.DisableResourceTypeDetection = true; });

```

Comment:
I don't think you should need the braces here... assignments can still be expression lambdas.